woocommerce/woocommerce
# Remove Creative Mail from WooCommerce defaults.

### Changes proposed in this Pull Request:

Closes #40966.

Removing defaults for Creative Mail extension as a part of the request to remove Creative Mail extensions from recommendations and WooCommerce task list.

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:


_Being at `develop` or `trunk` branch follow the steps below:_

1. Go to WooCommerce Settings and make sure to disable suggestions from WooCommerce.com website.

<img width="1280" alt="WooCommerce_settings_‹_WordPress_Pinterest_—_WordPress-2" src="https://github.com/woocommerce/woocommerce/assets/9010963/55438693-3a90-4d83-9fe9-8e77aab2be49">

## If it is already _OFF_ then turn in _ON_ and check steps _3._ and _4._ first before starting the test. We need to make sure we get Creative Mail from remote (WooCommerce.com) first before turning off the remote and checking local defaults.

2. Remove `_transient_woocommerce_admin_remote_free_extensions_specs` from `wp_options` database table.
3. Go to **WooCommerce** - **Home** and go to **You added sales channels**

<img width="1280" alt="Home_‹_WooCommerce_‹_WordPress_Pinterest_—_WooCommerce" src="https://github.com/woocommerce/woocommerce/assets/9010963/44acac71-145e-46ef-8371-1f0aea533cc4">

4. See the list of **Recommended marketing extensions**

<img width="1280" alt="Home_‹_WooCommerce_‹_WordPress_Pinterest_—_WooCommerce" src="https://github.com/woocommerce/woocommerce/assets/9010963/2cfc1473-2ea5-4204-93e2-73704df49533">

Now when we have suggestions from WooCommerce.com turned off and transient removed (it won't re-appear if remote suggestions are off) we get suggestions from local to WooCommerce extension's `DefaultFreeExtensions.php` class.

5. Checkout `remote/creative-mail` branch
6. Visit suggestions page again and check **Creative Mail for WooCommerce** is no longer there.

<img width="1280" alt="Home_‹_WooCommerce_‹_WordPress_Pinterest_—_WooCommerce" src="https://github.com/woocommerce/woocommerce/assets/9010963/ecf9ae81-f793-4531-80b9-b353c63523ac">

### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Remove Creative Mail for WooCommerce from recommended.

</details>
